### DISCUSSION TITLE
What's up with the alpha?

### DISCUSSION THREAD

**Discussion by akauppi** [firebase/firebase-admin-node#1450] [Q&A] [answered]
The `firebase-admin` alpha version has received no updates since a while, whereas the stable client was updated to 9.12.0.

1. Is moving in the direction of the alpha client still in the Firebase roadmap, or might it be abandoned?

2. npmjs.org [page](https://www.npmjs.com/package/firebase-admin/v/9.100.0-alpha.1) says the package would have been updated "10 days ago".
  
- Do you do new pushes without lifting the version number?

I have one project where I'm using the alpha, as a dependency. I am considering, whether I should move back to 9.12.0.

**Comment by hiranya911** [comment, marked as answer]
It will be released as v10 fairly soon. PR #1445 is already up, which will merge the modular-sdk branch into main. Then it will be released soon after.
